test/specs/point.spec.js

```diff
@@ -2,54 +2,77 @@ describe('Point annotation', function() {
   describe('auto', jasmine.fixtures('point'));
 
   describe('inRange', function() {
-    const annotation = {
+    const annotation1 = {
       type: 'point',
-      xValue: 5,
-      yValue: 5,
+      xValue: 7,
+      yValue: 7,
       radius: 30
     };
+    const annotation2 = {
+      type: 'point',
+      xValue: 3,
+      yValue: 3,
+      radius: 5
+    };
+    const annotation3 = {
+      type: 'point',
+      xValue: 3,
+      yValue: 3,
+      radius: 0
+    };
 
-    const chart = window.scatter10x10({test: annotation});
-    const element = window.getAnnotationElements(chart)[0];
+    const chart = window.scatter10x10({test1: annotation1, test2: annotation2, test3: annotation3});
```

Comment:
```suggestion
    const chart = window.scatter10x10({annotation1, annotation2, annotation3});
```

So the the id from error message is directly linked to annotation config rather than needing read the map (even thought numbers match, it still requires more processing from reader)

Reply:
done